contracts/VeLista.sol

```diff
@@ -186,12 +186,26 @@ contract VeLista is IVeLista, Initializable, AccessControlUpgradeable {
      */
     function increaseAmount(uint256 _amount) external {
         address _account = msg.sender;
+        _increaseAmountFor(_account, _amount, _account);
+    }
+
+    /**
+     * @dev increase lock amount for account; only compounder can call this function
+     * @param _account the account to increase the lock amount
+     * @param _amount amount of token to increase
+     */
+    function increaseAmountForCompound(address _account, uint256 _amount) external onlyRole(COMPOUNDER) {
+        address compounder = msg.sender;
+        _increaseAmountFor(_account, _amount, compounder);
+    }
+
+    function _increaseAmountFor(address _account, uint256 _amount, address _payer) internal {
```

Comment:
Per discussion, we'll remove access control for `increaseAmountFor` so that anyone can increase locked position for others by paying LISTA for them

Reply:
done in 83a986819f0970d3da70d8267414e4dece668e29